### ISSUE TITLE
The statistical information is too large. I hope it's like IOS

### ISSUE THREAD

**Issue by AlexKuai** [moonlight-stream/moonlight-android#1046]
If you use small screen devices such as mobile phones, the current statistics occupy too much area.

I hope it can be placed in the middle of the top of the screen like IOS, and it can be smaller. It's best to adjust the transparency.

Thank you.

**Comment by StarGinko** [comment]
> If you use small screen devices such as mobile phones, the current statistics occupy too much area.
> 
> I hope it can be placed in the middle of the top of the screen like IOS, and it can be smaller. It's best to adjust the transparency.
> 
> Thank you.

Good suggestion. I also perfer the statistical information in ios.

**Comment by AlexKuai** [comment]
> > If you use small screen devices such as mobile phones, the current statistics occupy too much area.
> > I hope it can be placed in the middle of the top of the screen like IOS, and it can be smaller. It's best to adjust the transparency.
> > Thank you.
> 
> Good suggestion. I also perfer the statistical information in ios.

Yes, on devices below 7 inches, statistics occupy too large a screen and cannot be adjusted.

**Comment by AlexKuai** [comment]
Resolution 2400 * 1080
